docs/rfc/0005_scan_filter.md

```diff
@@ -0,0 +1,113 @@
+|              |                                 |
+| :----------- | :------------------------------ |
+| Feature Name | Scan Filters                    |
+| Start Date   | Oct 27th, 2025                  |
+| Category     | Architecture                    |
+| RFC PR       | [#555](https://github.com/kubewarden/sbomscanner/pull/555), [#614](https://github.com/kubewarden/sbomscanner/issues/614) |
+| State        | **ACCEPTED**                    |
+
+# Summary
+
+[summary]: #summary
+
+Define a way to filter the images being scanned.
+
+# Motivation
+
+[motivation]: #motivation
+
+The purpose of this RFC is to define a way to filter certain parameters in order to speed up the scan time.
+Currently, when defining a Registry, the only way to limit the contents being scanned is to provide a list of repositories. However, all the images found inside these repositories are being scanned. This can lead to scanning a lot of irrelevant images, wasting a lot of time and resources.
+
+We should provide a way to filter the images being scanned.
+
+We would start focusing on two kinds of filters:
+
+* **platform**: many repositories contain multi-architecture images. A single image could have been built for linux as an OS, but for different architectures (arm64, amd64, s390x, ppc64, and maybe other flavors of arm). For example, as a user running a `x86_64` cluster, I care about scanning only amd64 images since these are the only ones I can run inside of my cluster.
+
+* **tags**: certain repositories do not remove old images. It would be useful to provide a way to scan only the images tagged in a specific way. As a possible solution, we might consider allowing the usage of a CEL expression to determine whether a tag is relevant or not. By using CEL, the end user would be able to use different methods: regular expressions, semantic versioning checks, etc.
+
+## Examples / User Stories
+
+[examples]: #examples
+
+- As a user, I want to scan only `linux/amd64` based images on my registry.
+- As a user, I want to scan `linux/amd64` and `linux/arm` based images on my registry.
+- As a user, I want to scan only the most recent tags of images on my registry.
+- As a user, I want to scan only the most recent tags, `linux/amd64` based images on my registry.
+- As a user, I want to scan only for non release candidate tags.
+- As a user, I want to scan for images deployed in a production environment with tags that end in `-prod`.
```

Comment:
Those stories look good, but I'd add a generic one to cover the overall filtering capability:

- As a user, I want to filter image tags using regular expressions, string comparisons, or semantic version operations.